buck/tox_hooks.py

```diff
@@ -229,18 +297,49 @@ def customize_envconfig(self, config):
             config.allowlist_externals = ["{toxinidir}/rename.sh",
                                           "charmcraft"]
 
+def get_gitreview_file():
```

Comment:
the .gitreview file is a .ini file format, there is a function that returns a reader object at https://github.com/freyes/buck/blob/main/buck/cmd.py#L22 , we can move that functionality to a "core.py" file or "utils.py" or "common.py"

Reply:
:+1: